### ISSUE TITLE
quick idea

### ISSUE THREAD

**Issue by prodzpod** [Zokonius/planar-artifice#65]
new cool boss -> summons eldritch crab as minions -> drops crab shell
-> make to craft dyable crimson armor

compact machine

**Comment by prodzpod** [comment]
a way to turn mobs into essentia

**Comment by Biviho** [comment]
hi! i don't know if this threads are open for discussions, but in case i will post what i think, maybe can help with development, or you can ignore them obviously

> a way to turn mobs into essentia

there should be already a way to do that in thaumcraft with liquid death, or you where thinking of something different?
If is something similar to lquid death i would suggest to find a way to avoid content overlap...something like a pistol(like the one in thaumic addition)/ a melee weapon / a machine (i personally would prefer the other two, since a machine is still similar to liquid death) that can be loaded with liquid death and if the mob is killed/recieve damage, it will store some essentia from that mob.
later the essentia stored can be retrived in a jar like with bottled essentia/pumped out if it is a machine

> universal theorizing: way to suck up theory/observation points and give you one of random category back in essentia/flux cost

a way to use theory/observation points in excess would be really cool, this would give the player a use for all that accumulated and usless point in the late game obtained with research and curiosities.

i would suggest something like a char where you can sit and munch on your brain to suck knowledge and create an item that when used will do what you wanted (so that a modpack dev can add other uses for this mechanic)

**Comment by prodzpod** [comment]
yeah liquid dead exists but i was talking moreso about some kind of machine utilizing it, probably with redstone inlays and stuff (inlays are pretty underutilized imo)

also some idea from years ago: 1/8 block & glass tools

**Comment by Biviho** [comment]
> yeah liquid dead exists but i was talking moreso about some kind of machine utilizing it, probably with redstone inlays and stuff (inlays are pretty underutilized imo)

usually farms with liquid death require just one bucket and then it's not used anymore (unless added as ingredient in a pack or used with immersive engineering chemical thrower).
Used in a farm is actually pretty strong, it kills mobs quickly, drop crystals and is not consumed, so to justifie the use as a consumable in a machine it would need some additional mechanics like unique drops in addition to crystals (maybe also configurable for packdevs)

maybe redstone inlays can be used to define the area of effect of the machine, or the machine is composed of multiple blocks that need redstone inlays to be connected
